src/pymax/mixins/scheduler.py

```
@@ -0,0 +1,28 @@
+
+class SchedulerMixin(ClientProtocol):
+    async def _run_periodic(
+        self, func: Callable[[], Any | Awaitable[Any]], interval: float
+    ) -> None:
+        while True:
+            try:
+                result = func()
+                if asyncio.iscoroutine(result):
+                    await result
+            except Exception as e:
+                tb = traceback.format_exc()
```

Ruff (0.14.8)

Local variable `tb` is assigned to but never used

Remove assignment to unused variable `tb`

(F841)